.changeset/hot-oranges-join.md

```diff
@@ -0,0 +1,19 @@
+---
+"@logto/experience-legacy": minor
+"@logto/integration-tests": minor
+"@logto/experience": minor
+"@logto/console": minor
+"@logto/phrases": minor
+"@logto/schemas": minor
+"@logto/core": minor
+---
+
+new MFA prompt policy
+
+You can now cutomize the MFA prompt policy in the Console:
+1. MfaPolicy.Mandatory: MFA is required for all users
+2. MfaPolicy.NoPrompt: Do not ask users to set up MFA - new
+3. MfaPolicy.PromptAtSignInAndSignUp: Ask users to set up MFA during registration (skippable, one-time prompt) - the old policy
```

Comment:
is it a rename only? if so, will it be better to just point it out explicitly because it doesn't affect the actual behavior?